build/thirdparty/darwin/Dockerfile

```diff
@@ -17,7 +17,7 @@
 # - kubectl (fetch)
 # - etcd (fetch)
 
-FROM golang:1.18 as builder
+FROM golang:1.19 as builder
```

Comment:
this one we cannot update yet
I think would be better we wait C+R and the other projects as usual update and then we begin to update the whole project to work with go 1.19

Reply:
Unfortunately the build will fail without this update

```
Step 12/22 : RUN make WHAT=cmd/kube-apiserver &&     cp _output/local/bin/${KUBE_BUILD_PLATFORMS}/kube-apiserver $DEST
 ---> Running in 7fc7e0c467a9

Detected go version: go version go1.18.5 linux/amd64.
Kubernetes requires go1.19 or greater.
Please install go1.19 or later.

!!! [0906 14:56:06] Call tree:
!!! [0906 14:56:06]  1: /kubernetes/hack/lib/golang.sh:822 kube::golang::setup_env(...)
!!! [0906 14:56:06]  2: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
!!! [0906 14:56:06] Call tree:
!!! [0906 14:56:06]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
Makefile.generated_files:61: .make/go-pkgdeps.mk: No such file or directory
make[1]: *** [Makefile.generated_files:73: .make/go-pkgdeps.mk] Error 1
make: *** [Makefile:528: generated_files] Error 2
```